python/app.py

```diff
@@ -0,0 +1,366 @@
+# vim: set fileencodings=utf-8
+# -*- coding: utf-8; -*-
+#
+# Licensed to CRATE Technology GmbH ("Crate") under one or more contributor
+# license agreements.  See the NOTICE file distributed with this work for
+# additional information regarding copyright ownership.  Crate licenses
+# this file to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# However, if you have executed another commercial license agreement
+# with Crate these terms will supersede the license and you may use the
+# software solely pursuant to the terms of the relevant commercial agreement.
+
+import json
+import uuid
+import base64
+import hashlib
+
+from datetime import datetime
+
+from urllib.request import urlopen, Request
+from urllib.error import HTTPError
+
+from crate.client import connect
+
+from flask import (
+    Flask,
+    g as app_globals,
+    make_response,
+)
+from flask.ext.restful import Api, Resource, reqparse
+
+
+# app configuration
+MAX_CONTENT_LENGTH = 1024 * 1024  # 1mb
+CRATE_HOST = 'localhost:4200'
```

Comment:
we are having a config file in php. don't you also want to put that stuff there?